projects/ion/src/lib/icon/svgs/iconsText/localization.ts

```diff
@@ -0,0 +1,2 @@
+export default `<path fill-rule="evenodd" clip-rule="evenodd" d="M8.66586 5.6675C8.66586 7.13977 7.47235 8.33328 6.00008 8.33328C4.52781 8.33328 3.3343 7.13977 3.3343 5.6675C3.3343 4.19523 4.52781 3.00172 6.00008 3.00172C7.47235 3.00172 8.66586 4.19523 8.66586 5.6675ZM7.33297 5.6675C7.33297 6.40364 6.73622 7.00039 6.00008 7.00039C5.26395 7.00039 4.66719 6.40364 4.66719 5.6675C4.66719 4.93137 5.26395 4.33461 6.00008 4.33461C6.73622 4.33461 7.33297 4.93137 7.33297 5.6675Z" fill="#8D93A5"/>
```

Comment:
```suggestion
export default `<path fill-rule="evenodd" clip-rule="evenodd" d="M8.66586 5.6675C8.66586 7.13977 7.47235 8.33328 6.00008 8.33328C4.52781 8.33328 3.3343 7.13977 3.3343 5.6675C3.3343 4.19523 4.52781 3.00172 6.00008 3.00172C7.47235 3.00172 8.66586 4.19523 8.66586 5.6675ZM7.33297 5.6675C7.33297 6.40364 6.73622 7.00039 6.00008 7.00039C5.26395 7.00039 4.66719 6.40364 4.66719 5.6675C4.66719 4.93137 5.26395 4.33461 6.00008 4.33461C6.73622 4.33461 7.33297 4.93137 7.33297 5.6675Z"/>
```
You'll need to remove fills.